tests/constraint/test_CollisionFree.cpp

```diff
@@ -64,6 +65,11 @@ class CollisionFreeTest : public ::testing::Test
     mStateSpace = std::make_shared<MetaSkeletonStateSpace>(group);
     mManipulator->enableSelfCollisionCheck();
     mManipulator->enableAdjacentBodyCheck();
+
+    // See what CollisionFreeOutcome's toString returns when no collision
```

Comment:
We should take advantage of having programmatic access to the information (rather than as a giant string). Maybe we can instead have a vector of body names that we check the outcome's vectors against.